python/lsst/ts/electrometer/controller.py

```diff
@@ -375,3 +377,9 @@ async def get_integration_time(self):
             f"{self.commands.get_integration_time(self.mode)}", has_reply=True
         )
         self.integration_time = float(res)
+
+    async def get_intensity(self):
```

Comment:
I strongly suggest you eliminate `self.last_value` and simply return the value from `get_intensity` because:
* You aren't using it anywhere except to report it here.
* It duplicates the information stored in self.csc.evt_intensity. So if you did want to get the most recently read value you can read it from there. This avoids having two copies that can get out of sync with each other.

Reply:
Please add a doc string.

Reply:
I added the doc string and will leave it to Eric to eliminate `self.last_value` or not.